library/common/glfw/embedder.cc

```diff
@@ -26,6 +27,20 @@
 #include "library/common/glfw/text_input_plugin.h"
 #include "library/common/internal/plugin_handler.h"
 
+#ifdef __linux__
```

Comment:
Can you add a comment explaining why these includes are here? (If the dependencies change in the future it's nice to have a reminder to clean this up.)

Reply:
Done.